lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -166,11 +168,11 @@ def define
             if calculator.going_abroad
               if calculator.country == "ireland"
                 question :is_british_or_irish?
-              elsif calculator.eea_country?
+              elsif calculator.eea_country? && calculator.country != "gibraltar"
```

Comment:
This looks like it may need a test..? It's a bit hard to tell, as all the tests are bunched together in a commit at the end, so I'm not sure if this change is covered. Fixing up the relevant test to the relevant commit would help signify the intent better.